.github/workflows/on-pullrequest.yml

```diff
@@ -112,6 +116,49 @@ jobs:
       - name: Linter
         run: npm run test:lint
 
+  upgrade:
+    # Upgrade packages on PR only to avoid a double update when pushed to main
+    name: Upgrade packages
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: Meadowlark-js
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout the Repo
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          fetch-depth: 0w
+
+      - name: Get changes
+        id: changes
+        run: |
+          suggested=v$(cat lerna.json | jq -r .version)
```

Comment:
To check weather or not we need to add a new commit for the new version, we compare the value in lerna.json against the latest tag available, if it's different, we assume that a change has already be done as part of this PR.

**warning** this means that manually updating the lerna.json version (not recommended) will cause the update not to run

Reply:
@bradbanister FYI